services/src/main/java/org/jboss/windup/web/services/rest/MigrationProjectEndpointImpl.java

```diff
@@ -44,15 +43,16 @@
     private WebPathUtil webPathUtil;
 
     @Override
-    public List<MigrationProjectAndAppCount> getMigrationProjects()
+    public List<MigrationProjectAndAppCount> getMigrationProjects(Boolean includeProvisional)
     {
         try
         {
             final String query =
                     "SELECT project, COUNT(DISTINCT app) AS appCount "
-                    + "FROM " + MigrationProject.class.getSimpleName() + " project "
-                    + "LEFT JOIN project.applications AS app "
-                    + "GROUP BY project.id";
+                    + "\nFROM " + MigrationProject.class.getSimpleName() + " AS project "
+                    + "\nLEFT JOIN project.applications AS app "
+                    + (Boolean.TRUE.equals(includeProvisional) ? "" : "\nWHERE project.provisional = FALSE")
+                    + "\nGROUP BY project.id";
```

Comment:
@klinki well, I guess, @OndraZizka is trying to have it readable in logs, but special space would be good ot have there too